code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -370,7 +369,19 @@ obj/item/weapon/material/hatchet/tacknife/combatknife/fluff/katarina/handle_shie
 
 	icon_override = 'icons/vore/custom_clothes_vr.dmi'
 	item_state = "serdyhelm_mob"
-*/
+
+//SilencedMP5A5:Serdykov Antoz
+/obj/item/device/modkit_conversion/fluff/serdykit
+	name = "Serdykov's hardsuit modification kit"
```

Comment:
It's not a hardsuit modification kit, but apparently just regular armor. Should be **"Serdykov's body armor customization kit"**